src/features/comments.ts

```diff
@@ -0,0 +1,74 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import { createComment, deleteComment, getPostComments } from '../api/comments';
+import { Comment } from '../types/Comment';
+import { Post } from '../types/Post';
+
+type CommentsState = {
+  comments: Comment[];
+  loaded: boolean;
+  hasError: boolean;
+};
+
+const initialState: CommentsState = {
+  comments: [],
+  loaded: false,
+  hasError: false,
+};
+
+export const fetchComments = createAsyncThunk<Comment[], Post['id']>(
+  'comments/fetch',
+  postId => getPostComments(postId),
+);
+
+export const addComment = createAsyncThunk<Comment, Omit<Comment, 'id'>>(
+  'comments/add',
+  comment => createComment(comment),
+);
+
+export const removeComment = createAsyncThunk<number, Comment['id']>(
+  'comments/remove',
+  async commentId => {
+    await deleteComment(commentId);
+
+    return commentId;
+  },
+);
```

Comment:
In your async thunks, you are not handling potential API errors. If an error occurs during the API call, it will cause an unhandled promise rejection. Consider adding try-catch blocks to handle these errors and dispatch a rejected action with the error message.